R/mmrm.R

```diff
@@ -204,10 +253,34 @@ fit_mmrm <- function(
 }
 
 
-#' Title
+
+#' Fit an mmrm model via multiple optimizers
+#'
+#'
+#' The function attempts to fit a mmrm model using the optimizer as specified in `optimizer`
+#' If `optimizer` is of length > 1 then it will loop through all the values until one of them is able
+#' to converge. That is to say if a fit fails to converge it will move onto the next value of `optimizer` and
+#' try again.
+#'
+#' If `optimizer` is a list then the names of the list with taken to be the required `optimizer`
```

Comment:
"If `optimizer` is a list then the names of the list with taken to be the required `optimizer`" sorry is this correct grammatically? I don't understand what you mean :)

Reply:
aww sorry "with" should be "will be" i.e.

"names of the list **will be** taken to be the

Reply:
```suggestion
#' If `optimizer` is a list then the names of the list will be taken to be the required `optimizer`
```